packages/api/src/account/queries/get_feature_flags.sql

```diff
@@ -0,0 +1,6 @@
+SELECT
```

Comment:
This query should live under /feature_flag/queries rather than here, since it's dealing with feature flags at the global level rather than at the account level